doc-content/shared-kie-docs/src/main/asciidoc/KieServer/securing-passwords-proc.adoc

```diff
@@ -0,0 +1,95 @@
+[id='securing-passwords-proc_{context}']
+
+= Securing {KIE_SERVER} passwords with a keystore
+
+{KIE_SERVER} uses passwords to authenticate users. For increased security, you can store {KIE_SERVER} passwords in a keystore. Because it supports symmetrics keys, you must use a Java Cryptography Extension KeyStore (JCEKS) with {KIE_SERVER}. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
+
+[NOTE]
+====
+If {KIE_SERVER} is not configured with JCEKS, {KIE_SERVER} stores passwords in configuration parameters in unsecured text files.
+====
+
+.Prerequisites
+
+* {KIE_SERVER} is installed in {EAP}.
```

Comment:
So my understanding, and what I've done to date, is everywhere we say EAP in enterprise, say Wildfly in community. EAP is an enterprise version of Wildfly, or at least the enterprise alternative, if I recall. Would be good to confirm with SMEs and then conditionalize accordingly. This would impact a lot of these instructions and snippets, so if it's true that we should conditionalize, we should probably create doc attributes accordingly.

Reply:
Yes, EAP is an enterprise version of WildFly.